pandas/core/indexes/range.py

```diff
@@ -40,7 +40,7 @@ class RangeIndex(Int64Index):
 
     Parameters
     ----------
-    start : int (default: 0), or other RangeIndex instance
+    start : int (default: 0), range or RangeIndex instance
```

Comment:
we didn’t do this originally because of @shoyer objections about too much inferences

Reply:
In the current implementation it is placed in a very simple if-statement and the whole ``__new__`` constructor is quite small, so IMO this would not be a burden for maintainability. But obviously not a huegly important issue.

Reply:
This has been removed again.